open_city_profile/settings.py

```diff
@@ -141,19 +141,15 @@
 ENABLE_GRAPHIQL = env("ENABLE_GRAPHIQL")
 
 INSTALLED_APPS = [
-    "helusers",
-    "helusers.providers.helsinki_oidc",
-    "django.contrib.admin",
+    "helusers.apps.HelusersConfig",
+    "helusers.apps.HelusersAdminConfig",
```

Comment:
A curiosity, but it might be possible to leave this out entirely, if we are not using helsinki-tunnistus for authentication into the admin.

Reply:
Oh. So what do those apps actually do then? I thought they add something into the admin interface.